practices/graded-browser-support.md

```diff
@@ -1,67 +1,90 @@
-# Graded browser support
+# Browser support
 
-* [Our criteria for browser support](#our-criteria-for-browser-support)
-* [How we implement graded browser support](#how-we-implement-graded-browser-support)
+This page describes the way that we support different browsers and browser versions when building sites at Springer Nature.
 
+* [Our criteria for browser support](#Our-criteria-for-browser-support)
+* [Graded browser support list](#Graded-browser-support-list)
+* [Implementing graded browser support](#Implementing-graded-browser-support)
+  * [Implementation details](#Implementation-details)
+  * [Caveats](#Caveats)
+    * [Internet Explorer 10 support](#Internet-Explorer-10-support)
+    * [Browsers without TLS 1.2 support](#Browsers-without-TLS-1.2-support)
 
 ## Our criteria for browser support
 
-We follow the guidelines for graded browser support outlined by Nate Koechley at Yahoo:
-[Yahoo Grade Browser Support guide](https://github.com/yui/yui3/wiki/Graded-Browser-Support)
+We follow the guidelines for graded browser support outlined by Yahoo in their [Graded Browser Support guide](https://github.com/yui/yui3/wiki/Graded-Browser-Support).
 
-Our browser support in summary:
+The graded browser support works by classifying all browser versions in one of three different grades:
 
-- Grade-A are modern and/or popular browser versions that receive all styling and behaviour. Bugs are addressed with high priority.
-- Grade-C are old and/or uncommon browser versions which are known to break if receiving full styling and behaviour. We serve only minimum CSS and JavaScript (detailed below). Bugs are addressed with high priority.
-- Grade-X browsers exist in the grey area between Grade-A and Grade-C. We serve them the same code as Grade-A, but we do not test our products against these browser versions. We expect our sites to work in these browsers. Over time, analysis of errors and usage patterns may cause some browsers in Grade-X to be deliberately changed to either Grade-A or Grade-C, to allow for better support. 
+* Grade-A are capable or very popular browser versions. These browsers will receive the full experience, with all styling and behaviour. Bugs are addressed with high priority.
+* Grade-C are incapable and/or uncommon browser versions which are known to break if receiving full styling and behaviour. These browsers will receive a core experience, with only minimum CSS and JavaScript. Bugs are addressed with high priority.
```

Comment:
Maybe add something about their relation to web standards "These incapable and/or uncommon browser versions do not support the latest web standards and are known to break if receiving full styling and behaviour.

Do we address Grade-C bugs with high priority? Does it perhaps mean "Bugs in the core experience are addressed with high priority"?

Reply:
> Maybe add something about their relation to web standards "These incapable and/or uncommon browser versions do not support the latest web standards and are known to break if receiving full styling and behaviour.

Ooh, I like that. Although maybe not say "are known to break" as usually when we classify a browser as Grade C is because someone wanted to do or use something specific, and that browser version breaks when using that thing. That said, I'm not sure what to replace it with.

> Do we address Grade-C bugs with high priority? Does it perhaps mean "Bugs in the core experience are addressed with high priority"?

Ah, this is one of these things that has been copied and pasted from document to document to document and it has lost its meaning. My understanding is that:

* Grade A: We test on them, we fix bugs that affect them with high priority.
* Grade C: We test on them, we fix bugs that affect them with high priority.
* Grade X: We don't test on them, we fix bugs that affect them with normal priority.

@dotcode Do you mind confirming if my understanding is correct? ^

We could probably explain that in a slightly better way.

Reply:
@josebolos your understanding is correct.